src/resources/subscription-manager-nose-tests.sh

```diff
@@ -15,10 +15,17 @@
 
 echo "sha1:" "${sha1}"
 
+# Decide which package manager to use
+if [ -a $(which $PM) ]; then
```

Comment:
Shouldn't be here `if [ -a $(which dnf) ]; then` and on the following line: `PM=dnf`? I cannot find anywhere that $PM has some default value. Maybe I miss something, maybe Jenkins set such environment variable. In that case there should be link to documentation with explanation.